### ISSUE TITLE
Unable to reproduce the ANN result of VGG16 On Cifar100.

### ISSUE THREAD

**Issue by bluehalk** [hzc1208/ANN2SNN_COS#1]
I'm unable to reproduce the same accuracy of VGG16 on Cifar100. Could you please provide me with the details?
My configurations are as follows:
learning rate (lr) = 0.02, epochs = 300, scheduler = cosine, optimizer = SGD with momentum 0.9, GPU: GeForce RTX 3090

**Comment by bluehalk** [comment]
When training an ANN, have you used the pretrained-model provided by the timm? If possible, could you provide the .pth file or your configs?
